media-gfx/shotwell/shotwell-0.30.4.ebuild

```diff
@@ -15,41 +15,39 @@ SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
 IUSE="opencv udev"
 
-COMMON_DEPEND="
-	>=x11-libs/gtk+-3.22.0:3
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	$(vala_depend)
+	>=app-crypt/gcr-3:=[gtk,vala]
+	>=dev-db/sqlite-3.5.9:3
+	dev-libs/appstream-glib
 	>=dev-libs/glib-2.40.0:2
-	>=dev-libs/libgee-0.8.5:0.8
-	>=net-libs/webkit-gtk-2.4:4
 	>=dev-libs/json-glib-0.7.6
+	dev-libs/libgdata
+	>=dev-libs/libgee-0.8.5:0.8
 	>=dev-libs/libxml2-2.6.32:2
-	x11-libs/gdk-pixbuf:2
-	>=dev-db/sqlite-3.5.9:3
-	media-libs/gstreamer:1.0
+	>=media-libs/gexiv2-0.10.4[vala]
```

Comment:
This, libsoup and more were on purpose doubled in DEPEND, without being in RDEPEND. We only need USE=vala during build for the vapi files, but not anymore at runtime (vala is "compiled" into C and then really compiled, afterall). So this was constructed by me on purpose to allow only having vala during build and not later. This can be useful for binary packages - you only need the vala USE flag while creating the binary packages, but then won't need when installing it on a system.
Your restructuring puts them into RDEPEND too, if I'm reading this right.

Reply:
Yep,  you are right! Fixing :)